pgmpy/readwrite/BIF.py

```diff
@@ -0,0 +1,564 @@
+from pgmpy.models import BayesianModel
+
+from pgmpy.factors import TabularCPD
+
+from pgmpy.extern.six.moves import map, range
+
+import numpy
+
+from pyparsing import Word, alphanums, Suppress, Optional, CharsNotIn, Group, nums, ZeroOrMore, OneOrMore,\
+        cppStyleComment, Literal, printables
+
+import re
+
+from string import Template
```

Comment:
@ankurankan You asked me to leave a blank line after imports (pep8 style) so left some.